.github/workflows/preview-env.yml

```diff
@@ -68,9 +68,6 @@ jobs:
           )
           IFS=, ; echo "env=${variables[*]}" >> $GITHUB_OUTPUT
 
-      - name: Create files volume
```

Comment:
**question**

why it is not needed anymore?

Reply:
We need a simpler way to handle volumes for preview environments, I'm just simplifying the flow for now